source/core/Window.cpp

```diff
@@ -16,18 +16,15 @@
 #include "Window.hpp"
 
 void Window::open(const std::string &caption, u16 width, u16 height) {
-	SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 2);
-	SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 1);
+	sf::ContextSettings settings(/*depthBits:*/24, /*stencilBits:*/8, /*antialiasingLevel:*/4);
```

Comment:
The comments here are a little bit too much

Reply:
Okay, better now?